ARCs/arc-0056.md

```diff
@@ -167,8 +167,20 @@ interface Method {
     name?: string;
     /** Optional, user-friendly description for the argument */
     desc?: string;
-    /** The default value that clients should use. MUST be base64 encoded bytes */
-    defaultValue?: string;
+    /** The default value that clients should use. */
+    defaultValue?: {
+      /** Base64 encoded bytes or uint64 */
+      data: string | bigint;
+      /** How the data is encoded. This is the encoding for the data provided here, not the arg type */
+      type: ABIType | AVMBytes;
```

Comment:
This should probably have an `AVMUInt64` and is required to be that when the data is `bigint` and otherwise can't be that?

Reply:
Yeah I'm realizing we need to have AVMUint64 for state keys/values as well. Initially I was thinking it could always just be a native uint64 but there are cases where an app creator might reserve max bytes schema since you can store uint64 as bytes but not always the other way around. I will add AVMUint64 here and in state as well.

Reply:
I've added `AVMUint64` and `AVMType` as a uinion of all the AVM types and used `AVMType` where appropriate